compile.sh

```diff
@@ -17,6 +17,7 @@ fpc -O3 fpc/code.pas
 crystal build -o crystal/code --release crystal/code.cr
 #gnatmake -O3 -gnat2022 -gnatp -flto ada/code.adb -D ada -o ada/code
 scala-cli --power package scala/code.scala -f -o scala/code
+scala-cli --power package scala/code.scala -f -o scala/code-native
```

Comment:
this line is literally identical to above line except output name; it will not build as scala-native binary. also, it is built with no optimization; [to optimize, the flag `--native-mode` is needed.](https://scala-native.org/en/stable/user/sbt.html#compilation-modes) 

```suggestion
scala-cli --power package --native scala/code.scala -f -o scala/code-native --native-mode release-full
```